{"asctime": "2022-02-28T12:49:59Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:49:59Z-0300"}
{"asctime": "2022-02-28T12:49:59Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:49:59Z-0300"}
{"asctime": "2022-02-28T12:49:59Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"75a6872a85a521d02744ce3998f7cfecbcbcfbf0\"\nCookie: _xsrf=2|ae77b744|a90816d3c4afc183246631019caf938e|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|ae77b744|a90816d3c4afc183246631019caf938e|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:49:59Z-0300"}
{"asctime": "2022-02-28T12:49:59Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:49:59Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c1f3319b-6ceb-48bf-92c6-7da74f4e5cd4", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 267, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:02Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 277, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "f67893e8-9978-4f4f-8d86-b7dec6735307", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:02Z-0300"}
{"asctime": "2022-02-28T12:50:04Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:04Z-0300"}
{"asctime": "2022-02-28T12:50:04Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"15e5a1ee564b946de734b667150e08c89529172c\"\nCookie: _xsrf=2|ae77b744|a90816d3c4afc183246631019caf938e|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:04Z-0300"}
{"asctime": "2022-02-28T12:50:04Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c1f3319b-6ceb-48bf-92c6-7da74f4e5cd4", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:04Z-0300"}
{"asctime": "2022-02-28T12:50:07Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:07Z-0300"}
{"asctime": "2022-02-28T12:50:07Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c1f3319b-6ceb-48bf-92c6-7da74f4e5cd4", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:07Z-0300"}
{"asctime": "2022-02-28T12:50:07Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c1f3319b-6ceb-48bf-92c6-7da74f4e5cd4", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:07Z-0300"}
{"asctime": "2022-02-28T12:50:08Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "1175b53d-403b-4e23-ac93-546ea988c8ef", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:08Z-0300"}
{"asctime": "2022-02-28T12:50:08Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:08Z-0300"}
{"asctime": "2022-02-28T12:50:08Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"07f4a9b3e82792912f8f9973120bcd6533b0448c\"\nCookie: _xsrf=2|13ff5cfe|1480fd6979272a3999eedabb21277834|1645816505; consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|13ff5cfe|1480fd6979272a3999eedabb21277834|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:08Z-0300"}
{"asctime": "2022-02-28T12:50:08Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "1175b53d-403b-4e23-ac93-546ea988c8ef", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:08Z-0300"}
{"asctime": "2022-02-28T12:50:11Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:11Z-0300"}
{"asctime": "2022-02-28T12:50:11Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "1175b53d-403b-4e23-ac93-546ea988c8ef", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:11Z-0300"}
{"asctime": "2022-02-28T12:50:11Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "1175b53d-403b-4e23-ac93-546ea988c8ef", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:11Z-0300"}
{"asctime": "2022-02-28T12:50:12Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:12Z-0300"}
{"asctime": "2022-02-28T12:50:12Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:12Z-0300"}
{"asctime": "2022-02-28T12:50:12Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "8aa14e49-e88a-4a31-847f-c85e13faf9d3", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:12Z-0300"}
{"asctime": "2022-02-28T12:50:13Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:13Z-0300"}
{"asctime": "2022-02-28T12:50:13Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:13Z-0300"}
{"asctime": "2022-02-28T12:50:13Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"f5a01c52b49d302289ae74723a62cbb635b30e17\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:13Z-0300"}
{"asctime": "2022-02-28T12:50:13Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:13Z-0300"}
{"asctime": "2022-02-28T12:50:16Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:16Z-0300"}
{"asctime": "2022-02-28T12:50:16Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:16Z-0300"}
{"asctime": "2022-02-28T12:50:16Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:16Z-0300"}
{"asctime": "2022-02-28T12:50:16Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 30, "module": "account", "threadName": "MainThread", "message": "Retrieving hub account information", "function": "HubHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:16Z-0300"}
{"asctime": "2022-02-28T12:50:16Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 40, "module": "account", "threadName": "MainThread", "message": "Retrieving hub account information", "function": "HubHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "73468130-443b-41ac-82b0-2094a9d39853", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:16Z-0300"}
{"asctime": "2022-02-28T12:50:17Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:17Z-0300"}
{"asctime": "2022-02-28T12:50:18Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"4cac4c43d0c3c3266e6dc1c3307a3f42e342b3c2\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:18Z-0300"}
{"asctime": "2022-02-28T12:50:18Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:18Z-0300"}
{"asctime": "2022-02-28T12:50:20Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:20Z-0300"}
{"asctime": "2022-02-28T12:50:20Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:20Z-0300"}
{"asctime": "2022-02-28T12:50:20Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:20Z-0300"}
{"asctime": "2022-02-28T12:50:20Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 148, "module": "account", "threadName": "MainThread", "message": "Retrieving spoke information", "function": "SpokeHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:20Z-0300"}
{"asctime": "2022-02-28T12:50:20Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 158, "module": "account", "threadName": "MainThread", "message": "Retrieving spoke information", "function": "SpokeHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "53374bcb-220c-4a73-8614-d912c2a9bf4b", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:20Z-0300"}
{"asctime": "2022-02-28T12:50:22Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:22Z-0300"}
{"asctime": "2022-02-28T12:50:22Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"b9260b6b91893e8126c3ea86df700d52d6debe0f\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:22Z-0300"}
{"asctime": "2022-02-28T12:50:22Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "8aa14e49-e88a-4a31-847f-c85e13faf9d3", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:22Z-0300"}
{"asctime": "2022-02-28T12:50:26Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:26Z-0300"}
{"asctime": "2022-02-28T12:50:26Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "8aa14e49-e88a-4a31-847f-c85e13faf9d3", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:26Z-0300"}
{"asctime": "2022-02-28T12:50:26Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "8aa14e49-e88a-4a31-847f-c85e13faf9d3", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:26Z-0300"}
{"asctime": "2022-02-28T12:50:28Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:28Z-0300"}
{"asctime": "2022-02-28T12:50:28Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: Cookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|d45aea70|d3254be7be829cb75e4b6c35e682ceba|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:28Z-0300"}
{"asctime": "2022-02-28T12:50:28Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:28Z-0300"}
{"asctime": "2022-02-28T12:50:31Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:31Z-0300"}
{"asctime": "2022-02-28T12:50:31Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:31Z-0300"}
{"asctime": "2022-02-28T12:50:31Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:31Z-0300"}
{"asctime": "2022-02-28T12:50:31Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 267, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:31Z-0300"}
{"asctime": "2022-02-28T12:50:31Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 277, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "691897df-2a41-4f7d-bbe8-ef447a923e32", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T12:50:31Z-0300"}
{"asctime": "2022-02-28T13:23:30Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c2384e8f-a0f7-4daf-b861-96d4a4203664", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:30Z-0300"}
{"asctime": "2022-02-28T13:23:30Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:30Z-0300"}
{"asctime": "2022-02-28T13:23:30Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"3efc8010cc78f8b5cb68eb1aa3aeb1c0dd1fd651\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|e631c034|e14e61a38ce9b6f36c204671d4e9e4fe|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:30Z-0300"}
{"asctime": "2022-02-28T13:23:30Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c2384e8f-a0f7-4daf-b861-96d4a4203664", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:30Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c2384e8f-a0f7-4daf-b861-96d4a4203664", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v1/auth?redirect_url=https://localhost:3000/settings", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "c2384e8f-a0f7-4daf-b861-96d4a4203664", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "7a55c3ec-6f6c-4710-ba20-4cd5fe3dff78", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"07f4a9b3e82792912f8f9973120bcd6533b0448c\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|134ce0f3|1433416479949634995d66b62194c439|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|134ce0f3|1433416479949634995d66b62194c439|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:33Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "7a55c3ec-6f6c-4710-ba20-4cd5fe3dff78", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:33Z-0300"}
{"asctime": "2022-02-28T13:23:36Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:36Z-0300"}
{"asctime": "2022-02-28T13:23:36Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "7a55c3ec-6f6c-4710-ba20-4cd5fe3dff78", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:36Z-0300"}
{"asctime": "2022-02-28T13:23:36Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v2/user_profile", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "7a55c3ec-6f6c-4710-ba20-4cd5fe3dff78", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:36Z-0300"}
{"asctime": "2022-02-28T13:23:37Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:37Z-0300"}
{"asctime": "2022-02-28T13:23:37Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:37Z-0300"}
{"asctime": "2022-02-28T13:23:38Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:38Z-0300"}
{"asctime": "2022-02-28T13:23:38Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 439, "module": "base", "threadName": "MainThread", "message": "Incoming request", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "dcb19fc2-84bc-4e17-ad74-c1a53761d05d", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:38Z-0300"}
{"asctime": "2022-02-28T13:23:38Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:38Z-0300"}
{"asctime": "2022-02-28T13:23:38Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"f5a01c52b49d302289ae74723a62cbb635b30e17\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:38Z-0300"}
{"asctime": "2022-02-28T13:23:39Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:39Z-0300"}
{"asctime": "2022-02-28T13:23:42Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:42Z-0300"}
{"asctime": "2022-02-28T13:23:42Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:42Z-0300"}
{"asctime": "2022-02-28T13:23:42Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/hub", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:42Z-0300"}
{"asctime": "2022-02-28T13:23:42Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 30, "module": "account", "threadName": "MainThread", "message": "Retrieving hub account information", "function": "HubHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:42Z-0300"}
{"asctime": "2022-02-28T13:23:42Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 40, "module": "account", "threadName": "MainThread", "message": "Retrieving hub account information", "function": "HubHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "3907b6ac-e8b3-4ca0-b217-a4c90174a111", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:42Z-0300"}
{"asctime": "2022-02-28T13:23:44Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:44Z-0300"}
{"asctime": "2022-02-28T13:23:44Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"4cac4c43d0c3c3266e6dc1c3307a3f42e342b3c2\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:44Z-0300"}
{"asctime": "2022-02-28T13:23:44Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:44Z-0300"}
{"asctime": "2022-02-28T13:23:47Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:47Z-0300"}
{"asctime": "2022-02-28T13:23:47Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:47Z-0300"}
{"asctime": "2022-02-28T13:23:47Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/spoke", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:47Z-0300"}
{"asctime": "2022-02-28T13:23:47Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 148, "module": "account", "threadName": "MainThread", "message": "Retrieving spoke information", "function": "SpokeHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:47Z-0300"}
{"asctime": "2022-02-28T13:23:47Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 158, "module": "account", "threadName": "MainThread", "message": "Retrieving spoke information", "function": "SpokeHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "b405829d-afd6-4d4b-87b1-f1859285af01", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:47Z-0300"}
{"asctime": "2022-02-28T13:23:48Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:48Z-0300"}
{"asctime": "2022-02-28T13:23:48Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"75a6872a85a521d02744ce3998f7cfecbcbcfbf0\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:48Z-0300"}
{"asctime": "2022-02-28T13:23:48Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:48Z-0300"}
{"asctime": "2022-02-28T13:23:51Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:51Z-0300"}
{"asctime": "2022-02-28T13:23:51Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:51Z-0300"}
{"asctime": "2022-02-28T13:23:51Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/services/aws/account/org", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:51Z-0300"}
{"asctime": "2022-02-28T13:23:51Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 267, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:51Z-0300"}
{"asctime": "2022-02-28T13:23:51Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "account.py", "funcName": "get", "levelname": "DEBUG", "lineno": 277, "module": "account", "threadName": "MainThread", "message": "Retrieving org information", "function": "OrgHandler.get", "user": "user@noq.dev", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "81457c3b-26c1-4f9e-9454-9e3a43af0bb2", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:51Z-0300"}
{"asctime": "2022-02-28T13:23:53Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "auth.py", "funcName": "get_groups", "levelname": "ERROR", "lineno": 183, "module": "auth", "threadName": "MainThread", "message": "auth.get_groups not configured properly or no groups were obtained.", "exc_info": "NoneType: None", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:53Z-0300"}
{"asctime": "2022-02-28T13:23:53Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "exceptions.py", "funcName": "__init__", "levelname": "ERROR", "lineno": 14, "module": "exceptions", "threadName": "MainThread", "message": "Groups not detected. Headers: If-None-Match: \"b9260b6b91893e8126c3ea86df700d52d6debe0f\"\nCookie: consoleme_auth=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYmYiOjE2NDU4MTE5NjQsImlhdCI6MTY0NTgxMTk2OSwiZXhwIjoxNjQ2MTQ5ODA3LCJlbWFpbCI6InVzZXJAbm9xLmRldiIsImdyb3VwcyI6W10sImhvc3QiOiJsb2NhbGhvc3QifQ.a-dRLotoyf1MV-4skmV7l--Rlz6mlwcNfNli0ntHrKw; _xsrf=2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nAccept-Language: pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7\nAccept-Encoding: gzip, deflate, br\nReferer: https://localhost:3000/settings\nSec-Fetch-Dest: empty\nSec-Fetch-Mode: cors\nSec-Fetch-Site: same-origin\nSec-Ch-Ua-Platform: \"macOS\"\nX-Xsrftoken: 2|990996ef|9e763778f3d1e028131810aaabd1b225|1645816505\nX-Requested-With: XMLHttpRequest\nAccept: application/json\nContent-Type: application/json\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36\nSec-Ch-Ua-Mobile: ?0\nSec-Ch-Ua: \" Not A;Brand\";v=\"99\", \"Chromium\";v=\"98\", \"Google Chrome\";v=\"98\"\nConnection: close\nHost: localhost:3000\n", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:53Z-0300"}
{"asctime": "2022-02-28T13:23:53Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "ERROR", "lineno": 614, "module": "base", "threadName": "MainThread", "message": "No groups detected. Check configuration.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "dcb19fc2-84bc-4e17-ad74-c1a53761d05d", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:53Z-0300"}
{"asctime": "2022-02-28T13:23:55Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "__init__.py", "funcName": "retrieve_credential_authorization_mapping", "levelname": "ERROR", "lineno": 90, "module": "__init__", "threadName": "MainThread", "message": null, "function": "CredentialAuthorizationMapping.retrieve_credential_authorization_mapping", "error": "Error loading cloud credential mapping. Returning empty mapping: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "host": "localhost", "exc_info": "Traceback (most recent call last):\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cloud_credential_authorization_mapping/__init__.py\", line 77, in retrieve_credential_authorization_mapping\n    ] = await retrieve_json_data_from_redis_or_s3(\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/cache/__init__.py\", line 215, in retrieve_json_data_from_redis_or_s3\n    s3_object = get_object(Bucket=s3_bucket, Key=s3_key, host=host)\n  File \"/Users/cspilhere/projects/cloudumi/common/lib/s3_helpers/__init__.py\", line 158, in get_object\n    return client.get_object(Bucket=kwargs.get(\"Bucket\"), Key=kwargs.get(\"Key\"))\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 391, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Users/cspilhere/projects/cloudumi/env/lib/python3.9/site-packages/botocore/client.py\", line 719, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the GetObject operation: Access Denied", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:55Z-0300"}
{"asctime": "2022-02-28T13:23:55Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "WARNING", "lineno": 638, "module": "base", "threadName": "MainThread", "message": "No eligible roles detected for user. But letting them continue", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "dcb19fc2-84bc-4e17-ad74-c1a53761d05d", "host": "localhost", "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:55Z-0300"}
{"asctime": "2022-02-28T13:23:56Z-0300", "name": "cloudumi", "processName": "MainProcess", "filename": "base.py", "funcName": "authorization_flow", "levelname": "DEBUG", "lineno": 646, "module": "base", "threadName": "MainThread", "message": "Successfully authorized user.", "function": "Basehandler.authorization_flow", "ip": "127.0.0.1", "request_path": "/api/v3/integrations/aws", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36", "request_id": "dcb19fc2-84bc-4e17-ad74-c1a53761d05d", "host": "localhost", "eligible_roles": 0, "eligible_accounts": 0, "eventTime": "2022-02-25T09:59:27.884326-08:00", "hostname": "MacBook-Pro-de-Christian.local", "timestamp": "2022-02-28T13:23:56Z-0300"}
